fbs/common.fbs

```diff
@@ -0,0 +1,30 @@
+
+namespace FBS.Common;
+
+/* Common custom types */
+
+table StringString {
+  key:string(required);
+  value:string(required);
+}
+
+table StringUint8 {
+  key:string(required);
+  value:uint8;
+}
+
+table Uint16String {
+  key:uint16;
+  value:string(required);
+}
+
+table Uint32String {
+  key:uint32;
+  value:string(required);
+}
+
+table StringStringArray {
+  key:string (required);
```

Comment:
There is a space here not present in other (require) usages

Reply:
Yes, I' add a space everywhere.